tests/test__influx_reporter.py

```diff
@@ -21,20 +21,20 @@ def setUp(self):
     def tearDown(self):
         super(TestInfluxReporter, self).tearDown()
 
-    def test_report_now(self):
+    def test_not_called_on_blank(self):
         influx_reporter = InfluxReporter(registry=self.registry)
 
         with mock.patch("pyformance.reporters.influx.urlopen") as patch:
             influx_reporter.report_now()
-            patch.assert_called()
+            patch.assert_not_called()
 
     def test_create_database(self):
         r1 = InfluxReporter(registry=self.registry, autocreate_database=True)
         with mock.patch("pyformance.reporters.influx.urlopen") as patch:
             r1.report_now()
-            if patch.call_count != 2:
+            if patch.call_count != 1:
```

Comment:
I didn't understand why this needed to change from 2 to 1 - please explain (f2f or here if it's enough)

Reply:
it was sending an empty request right after creating a database, it was invalid but not considered bad enough to bother with.